packages/theme-predictive-search/README.md

```diff
@@ -1,43 +1,47 @@
 ## Shopify Predictive Search JS API
 
-### Configuration options 
+### Configuration options
 
 <table>
   <tr>
-    <th width="30%">Type of search</th>
-    <th>Options</th>
+    <th width="30%">Options</th>
+    <th>Type</th>
+    <th>Description</th>
   </tr>
   <tr>
-    <td><code>resources</code></td>
-    <td>
-      <table>
-        <tr>
-          <th width="30%">Option</th>
-          <th>Type</th>
-          <th>Description</th>
-        </tr>
-        <tr>
-          <td><code>types</code> (required)</td>
-          <td>Array</td>
-          <td>Specifies the type of results requested. Currently supports <code>product</code> only.</td>
-        </tr>
-        <tr>
-          <td><code>fuzzy</code> (optional)</td>
-          <td>Boolean</td>
-          <td>Enables typo tolerance when searching. A typo will be tolerated after the fourth character. Default: <code>true</code>.</td>
-        </tr>
-        <tr>
-          <td><code>unavailable_products</code> (optional)</td>
-          <td>String</td>
-          <td>Specifies whether to display <code>unavailable_products</code> results. The three possible options are <code>show</code>, <code>hide</code>, and <code>bury</code>. Burying out of stock products pushes them below other matching results.  Default: <code>show</code>.</td>
-        </tr>
-        <tr>
-          <td><code>limit</code> (optional)</td>
-          <td>Integer</td>
-          <td>Limits the number of results returned. Default: <code>10</code>. Min:  <code>1</code>. Max: <code>10</code>.</td>
-        </tr>
-      </table>
-    </td>
+    <td><code>q</code> (required)</td>
+    <td>String</td>
+    <td>The search query.</td>
+  </tr>
+  <tr>
+    <td><code>resources</code> (required)</td>
+    <td>Object</td>
+    <td>Requests <code>resources</code> results for the given query, based on the <code>type</code> and <code>limit</code> fields.</td>
+  </tr>
+  <tr>
+    <td><code>type</code> (required)</td>
+    <td>Comma-separated values</td>
```

Comment:
We say comma-separated values here, but this is what is needed when building the request URL. The Predictive Search Library can receive an array for `fields` and `type` as it automatically transforms arrays into comma-separated lists when building the request URL.

Reply:
Good catch 👍